Packs/SentryWire/Integrations/SentryWire/SentryWire.py

```diff
@@ -0,0 +1,370 @@
+        except KeyError:
+            pass
+    return response
+
+
+def parse_search_status(response):
+    # Search was cancelled
+    if response["SearchResult"] == "Cancelled":
+        readable_output = "Search was cancelled"
+        response["SearchStatus"] = "Cancelled"
+        response["SearchResult"] = "NoPcapData"
+    # Search was completed with no pcap data
+    elif response["SearchResult"] == "NoPcapData":
+        readable_output = "Search completed: No results found"
+        response["SearchStatus"] = "Completed"
+    else:
+        # Default
+        readable_output = ""
+        try:
+            results = response["SearchResult"]
+            readable_output = f"Search completed: {results}"
+
+            response["SearchStatus"] = "Completed"
+            response["PcapSize"] = response["SearchResult"].split(" ")[-1].split(":")[-1]
+        except Exception:
+            raise DemistoException(
+                f'Could not get the size of SearchResult, got the following object: {response["SearchResult"]}')
+    return response, readable_output
+
+
+def parse_create_search(response):
+    try:
+        # Search ID handling
+        search_id = re.search("searchname=([^&]+)", response[0].get("checkstatus")).group(1)  # type: ignore[union-attr]
+
+        # Node Name Handling
+        outputs = {"NodeName": []}  # type: ignore[var-annotated]
+        node_str = ""
+        for i in range(len(response)):
+            node_name = re.search("nodename=([^&]+)", response[i].get("checkstatus")).group(1)  # type: ignore[union-attr]
+            outputs["NodeName"].append(node_name)
+            node_str += f"{node_name},"
+        node_str = node_str[:-1]
+        return search_id, outputs, node_str
+    except Exception:
+        raise DemistoException(f'Could not parse search_id from the following response: {response}')
+
+
+''' COMMANDS '''
+
+
+def create_search_command(client: Client, args: dict[str, Any]):
+    if args.get('generate_links'):
+        generate_links = argToBoolean(args.get('generate_links'))
+    else:
+        generate_links = False
+
+    response = client.create_search(args)
+
+    search_id, outputs, node_str = parse_create_search(response)
+
+    readable_output = f"SearchID : {search_id}\nNodeName(s) : {node_str}"
+
+    outputs["SearchID"] = search_id
+
+    if generate_links:
+        readable_output += "\n"
+        outputs["checkstatus"] = []
+        outputs["getpcaps"] = []
+        outputs["metadata"] = []
+        for i in range(len(response)):
+            for key in response[i]:
+                value = str(response[i].get(key))
+                readable_output += f"{key} : {value}\n"
+            readable_output = readable_output[:-1]
+            outputs["checkstatus"].append(response[i].get("checkstatus"))
+            outputs["getpcaps"].append(response[i].get("getpcaps"))
+            outputs["metadata"].append(response[i].get("metadata"))
+    return CommandResults(
```

Comment:
Since you are updating the response please all add a raw_respone to the CommandResults.